### DISCUSSION TITLE
New Installation with Ubuntu 18.04

### DISCUSSION THREAD

**Discussion by lexisman** [Ylianst/MeshCentral#5603] [Issues]
![image_2023-12-02_224642776](https://github.com/Ylianst/MeshCentral/assets/38063252/b75bcb22-2734-498f-a3ea-b9de67db9c81)
Please take a look on the image. Running node -v , shows version v14.4.0, however during the installation of meshcentral, warning that meshcentral need v11, and the bunch of error. Anyone can tell me what am I missing here. OS is ubuntu 18.04 on vm.

Sorry, I think I found the answer, updating nodejs also solve the issue. Thanks.

**Comment by dinger1986** [comment]
Why is the OS 18.04?

Can you try 20.04 or 22.04 or Debian 11 or 12.

**Comment by si458** [comment]
You are running node without sudo but npm with sudo

So I'm guessing, when npm is run with root its using a different version of node

Check the output of `sudo node -v`

Also note i recommend the latest stable LTS versions of node (18,20,22)

**Comment by lexisman** [comment]
Thanks.
